compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -99,6 +99,11 @@ static llvm::cl::opt<IREE::Codegen::WorkgroupId>
 
     );
 
+static llvm::cl::opt<bool>
+    clUseDirectLoad("iree-codegen-gpu-use-direct-load",
+                    llvm::cl::desc("Use global load DMA for direct load ops"),
+                    llvm::cl::Hidden, llvm::cl::init(false));
```

Comment:
I think we have a hidden convention is that this kind of flag should start with `iree-llvmgpu-*`. I don't always track llvmgpu flags, so some flags do not follow it. (I spent some time on cleaning these up for llvmcpu backends before.)

```suggestion
static llvm::cl::opt<bool>
    clUseDirectLoad("iree-llvmgpu-use-direct-load",
                    llvm::cl::desc("Use global load DMA for direct load ops."),
                    llvm::cl::Hidden, llvm::cl::init(false));
```